packages/tempo/bin/push-docs.sh

```
@@ -0,0 +1,49 @@
+  exit 1
+fi
+
+echo "Switching to main branch..."
+git checkout main
+
+echo "Pulling latest main..."
+git pull --ff-only origin main
+
+echo "Applying doc changes from $CURRENT_BRANCH to main..."
+# Checkout only the doc files from the branch
+git checkout $CURRENT_BRANCH -- $DOC_PATHS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)